SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetBitWriter.h

```diff
@@ -23,4 +24,6 @@ class SPATIALGDK_API FSpatialNetBitWriter : public FNetBitWriter
 	void SerializeObjectRef(FUnrealObjectRef& ObjectRef);
 
 	TSet<TWeakObjectPtr<const UObject>>& UnresolvedObjects;
+
+	USpatialNetDriver* NetDriver;
```

Comment:
I think this is a reason those catch all functions for resolve entities should not be in the NetDriver. We already are passing in the package map, lets use it instead.

Reply:
All the new functions have been moved into PackageMapClient